### ISSUE TITLE
In the Gauntlet tab, can you sort by % chance and then points?

### ISSUE THREAD

**Issue by TheMegaSage** [IAmPicard/StarTrekTimelinesSpreadsheet#216]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
In the Gauntlet tab, if multiple combinations have an identical chance, the sort appears to be by lowest point total first. If ten characters have 100% chance, and nine of them give 10 points and one gives 250 points, it would be nice to see the 250 points appear first in the list.

**Describe alternatives you've considered**
N/A

**Additional context**
Tool version: **0.8.1**
Operating system: **win32 x64 (6.1.7601)**

**Comment by PhantumPhysics** [comment]
I agree with previous comment, I always try for lowest points to get a longer gauntlet streak and better chest rewards.  I would prefer to see some sort of weighting for this the same way you can select weighting for the best point reward chance.  Only this would rank based on chance to win the smallest amount of points.  The difference between 99 and 97% success is not so much, and I would choose 95% chance of 10 points over 100% chance of 180 points early in gauntlet.  Then near the end switch to highest points to get a better rank.
